recipe/meta.yaml

```diff
@@ -27,9 +27,6 @@ requirements:
     - cython
     - numpy
     - libgdal
-    # pygeos is lagging behind on geos support
-    # we need it here to make sure the solver gets a compatible goes+gdal match
-    - pygeos
```

Comment:
@brendan-ward I removed it here from the "host" section as I think this is no longer needed for what is mentioned above (pygeos has had recent rebuilds)

But would you be OK to also remove it from the actual "run" dependencies below? It isn't an actual required dependency, and that would allow you to use it with geopandas and shapely 2.0 without automatically also installing pygeos.

Reply:
Yes, it can be removed.  I think it is only specifically needed in a few paces for testing.